rs/nns/governance/src/neuron_store.rs

```diff
@@ -740,29 +786,31 @@ impl NeuronStore {
     fn filter_map_active_neurons<R>(
         &self,
         filter: impl Fn(&Neuron) -> bool,
-        f: impl FnMut(&Neuron) -> R,
+        f: impl FnMut(Neuron) -> R,
     ) -> Vec<R> {
-        self.active_neurons_iter()
-            .filter(|n| filter(n))
-            .map(f)
-            .collect()
+        self.with_active_neurons_iter(|iter| iter.filter(|n| filter(n)).map(f).collect())
+    }
+
+    fn is_active_neurons_fund_neuron(neuron: &Neuron, now: u64) -> bool {
+        !neuron.is_inactive(now)
+            && neuron
+                .joined_community_fund_timestamp_seconds
+                .unwrap_or_default()
+                > 0
```

Comment:
I think this could be implemented inside an `impl Neuron`

Reply:
Changed to use Neuron's implementation of is_a_neurons_fund_member, but left function here, since it's just combining 2 different neuron methods.  Seems overly specialized to make a new method on the neuron to do that.